### PR TITLE
Add podspec file

### PR DESCRIPTION
**Pull request by buhe**
Awesome project, Thanks.

### CODE REVIEW

**Comment by yorkie** [comment]
@buhe this looks good, that would be graceful if you implements a generator/script to generate this file from "package.json" in npm "prepublish"?

Some guidance of this proposal:

1. write a script like the following:

    ```js
    const pkgInfo = require('package.json');
    const fs = require('fs');
    fs.writeFileSync('ios/RCTWeChat.podspec', '...build from pkgInfo...');
    ```

2. add the following to `package.json`:

    ```json
    "scripts": {
      "prepublish": "node ./scripts/generate-podspec.js"
    }
    ```

With the above steps, we can generate the file from package.json for npm users when publishing to npm :)

File: android/build.gradle [hunk 1]
```diff
@@ -14,8 +14,14 @@ android {
   }
 }
 
+allprojects {
+  repositories {
+    jcenter()
+    maven { url "$projectDir/../../react-native/android" }
```
Comment (yorkie) [inline android/build.gradle]: Why do you add this file? @buhe

Comment (buhe) [inline android/build.gradle]: @yorkie react-native  android hasn't published to maven , dependency by local dir.

Comment (buhe) [inline android/build.gradle]: like this : https://github.com/react-native-community/react-native-svg/blob/master/android/build.gradle

Comment (yorkie) [inline android/build.gradle]: Reasonable 👍

**Pull request merged**